src/ui/DynamicHierarchicalFacet/DynamicHierarchicalFacet.ts

```diff
@@ -239,6 +239,48 @@ export class DynamicHierarchicalFacet extends Component implements IDynamicHiera
       section: 'Sorting'
     }),
 
+    /**
+     * Specifies a custom order by which to sort the dynamic hierarchical facet values.
+     *
+     * **Example:**
+     * ```html
+     * <div
+     *   class="CoveoDynamicHierarchicalFacet"
+     *   data-field="@countries"
+     *   data-title="Countries"
+     *   data-tab="All"
+     *   data-clear-label="All Countries"
+     *   data-delimiting-character="|"
+     *   data-custom-sort="America, Asia|Japan, America|Mexico, America|Canada, Europe|Germany, Europe|France, Oceania"
+     * ></div>
+     * ```
+     */
+    customSort: ComponentOptions.buildStringOption({ section: 'Sorting' }),
```

Comment:
Is custom sort needed on the hierarchical facet too? The jira only mentions the regular dynamic facet?

If it's not needed now, I would defer adding the two options needed to make the feature work,

Reply:
I discussed this with Olivier and he said I should include it.